TheaterSchedule.BLL/Services/PerformanceScheduleService.cs

```diff
@@ -1,43 +1,45 @@
-using TheaterSchedule.DAL.Interfaces;
-using Microsoft.Extensions.Caching.Memory;
-using TheaterSchedule.BLL.DTO;
-using TheaterSchedule.BLL.Interfaces;
-
-namespace TheaterSchedule.BLL.Services
-{
-    public class PerformanceScheduleService : IPerformanceScheduleService
-    {
-        private IPerformanceScheduleRepository performanceScheduleRepository;
-        private IMemoryCache memoryCache;
-
-        public PerformanceScheduleService(IPerformanceScheduleRepository performanceScheduleRepository, IMemoryCache memoryCache)
-        {
-            this.performanceScheduleRepository = performanceScheduleRepository;
-            this.memoryCache = memoryCache;
-        }
-
-        public PerformanceScheduleDTO LoadScheduleData(int performanceId)
-        {
-            PerformanceScheduleDTO performanceScheduleRequest;
-            string memoryCacheKey = "Performance_" + performanceId.ToString();
-            var scheduleDataModel = performanceScheduleRepository.GetPerfomanceScheduleInfo(performanceId).Result;
-
-            if (!memoryCache.TryGetValue(memoryCache, out performanceScheduleRequest))
-            {
-                performanceScheduleRequest = new PerformanceScheduleDTO()
-                {
-                    PerformanceId = scheduleDataModel.PerformanceId,
-                    Title = scheduleDataModel.Title,
-                    MainImage = scheduleDataModel.MainImage,
-                    ScheduleList = scheduleDataModel.ScheduleList,
-                    Age = scheduleDataModel.Age,
-                    Duration = scheduleDataModel.Duration,
-                    Price = scheduleDataModel.Price,
-                };
-                memoryCache.Set(memoryCacheKey, performanceScheduleRequest);
-            }
-
-            return performanceScheduleRequest;
-        }
-    }
-}
+using TheaterSchedule.DAL.Interfaces;
+using Microsoft.Extensions.Caching.Memory;
+using TheaterSchedule.BLL.DTO;
+using TheaterSchedule.BLL.Interfaces;
+using TheaterSchedule.BLL.Helpers;
+
+namespace TheaterSchedule.BLL.Services
+{
+    public class PerformanceScheduleService : IPerformanceScheduleService
+    {
+        private IPerformanceScheduleRepository performanceScheduleRepository;
+        private ICacheProvider cacheProvider;
+
+        public PerformanceScheduleService(IPerformanceScheduleRepository performanceScheduleRepository, ICacheProvider cacheProvider)
+        {
+            this.performanceScheduleRepository = performanceScheduleRepository;
+            this.cacheProvider = cacheProvider;
+        }
+
+        public string GetMemoryCacheKey(int performanceId)
+        {
+            return string.Format("Performance_" + performanceId.ToString());

```

Comment:
Correct usage of string.format:
`string.Format("Performance_{0}", performanceId);` (ToString not needed, also don't use + in formatting templates)

Or use new syntax:
`return $"Performance_{performanceId}";`